sdk/communication/azure-communication-identity/src/main/java/com/azure/communication/identity/models/CommunicationTokenScope.java

```diff
@@ -10,12 +10,24 @@
 
 /** List of scopes for an access token. */
 public final class CommunicationTokenScope extends ExpandableStringEnum<CommunicationTokenScope> {
-    /** Static value chat for CommunicationTokenScope. */
+    /** Use this for full access to Chat APIs. */
     public static final CommunicationTokenScope CHAT = fromString("chat");
 
-    /** Static value voip for CommunicationTokenScope. */
+    /** Use this for full access to Calling APIs. */
     public static final CommunicationTokenScope VOIP = fromString("voip");
 
+    /** Access to Chat APIs but without the authorization to create, delete or update chat threads. */
+    public static final CommunicationTokenScope CHAT_JOIN = fromString("chat.join");
+
+    /**
+     * A more limited version of chat.join that doesn't allow to add or remove participants. Use this scope when the
+     * token bearer is not fully trusted, for example in guest scenarios.
```

Comment:
`Use this scope when the token bearer is not fully trusted, for example in guest scenarios.`
By ‘token bearer’, I believe you’re referring to  [Microsoft Entra ID ](https://learn.microsoft.com/en-us/azure/azure-app-configuration/rest-api-authentication-index#microsoft-entra-id) authentication, so let’s maintain consistency with our public documentation. I think we should provide a more detailed explanation to `not fully trusted`, good places can be README(for all platforms) file and the public docs, wdyt?

Reply:
Can you please elaborate what detailed explanation we need to provide? and I think we can have separate story for this documentation if needed.
The scope description is something we added in swagger so that it becomes clear for architects to review the usage of these new scopes.

Reply:
Wearing my Contoso Developer Advocate hat 🧙 will consider “Use this scope when the token bearer is not fully trusted, for example in guest scenarios.” statement, from a developer’s perspective, I might have a couple of questions:
1. Is the bearer token referring to Microsoft Entra ID authentication?
2. How can a Contoso developer determine if a token is fully trusted?

I guess, the statement was provided by you @DominikMe, can you please help us to understand the statement better, so we can provide detailed explanation in our public docs and readme file?😊

Reply:
@DominikMe Could you please assist with these questions so that we can provide detailed explanation in our documentation.